### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by krisbobo**
## Project: Mobile - menu ##
**The following changes were implemented for the mobile menu:**
- When the user clicks the hamburger button, the mobile menu appears.
- When the user clicks the close (X) button, the mobile menu disappears.
- When the user clicks any of the mobile menu options, the mobile menu disappears.
- When the user clicks any of the mobile menu options, a correct part of the page is displayed.
- When the user changes the viewport to desktop, the mobile menu disappears but reappears upon switching back to mobile.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const navToggle = document.querySelector('.nav-toggle');
+const navbar = document.querySelector('.mobile-nav-hidden');
+const closeHamburger = document.querySelector('.hidden');
+const hamburger = document.querySelector('#hamburger');
+const closeMenu = document.querySelector('.mobile-nav-menu');
+const navIcon = document.querySelectorAll('.navIcon');
+
+hamburger.addEventListener('click', () => {
+  navbar.classList.add('header');
```
Comment (koachcode) [inline js/main.js:9]: - [Optional] Please try to change the text color of the `language and framework` to `background: #9490FF;` as it is in the Figma design, currently, yours is `#000`


##### Your design

Screenshot from 2022-06-17 12-39-23

##### Figma design 
Screenshot from 2022-06-17 12-36-02

**Review by koachcode — APPROVED** [review]
Hi @krisbobo , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 1 above)
Comment (krisbobo) [reply js/main.js:9]: I'm not excactly sure how to go about that as it looks fine on my side. Possibly you could through more light on it...

Comment (krisbobo) [reply js/main.js:9]: Thanks, I couldn't see this because I had conflicts. Changes have been effected

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request ❌

Hi @krisbobo, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that is merged and deleted. Kindly make sure to create a new branch and then both of the members contribute to the repo, also don't submit the merged branch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**